std_msgs/Header header
float32[] eye_centers_3d_0
float32[] eye_centers_3d_1
float32[] gaze_normals_3d_0
float32[] gaze_normals_3d_1
float32[] gaze_point_3d
float32[] norm_pos
float32 confidence
float32 timestamp

